package.json

```diff
@@ -36,6 +36,7 @@
     "@types/node": "^18.14.0",
     "jest": "^29.4.3",
     "jsdoc": "^4.0.2",
+    "node-notifier": "^10.0.1",
```

Comment:
@harshithad0703 , Have you installed this package on purpose?

Reply:
yes, was getting `Error: notify reporter requires optional peer dependency "node-notifier" but it was not found`